WebRoot/css/zm.css

```diff
@@ -1975,14 +1975,48 @@ div.CompDragArea * {
 	display:inline-block;
 }
 
-.newTaskBanner {
-	@InactiveCursor@
-	color:rgb(120, 111, 91);
-	padding:5px 0px;
+div.newTaskBanner div.ImgAdd-svg {
+	@TaskListViewAddNewTaskIcon@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-ColHeader {
+	border-bottom:0px;
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble {
+	@TaskListViewItem@
+}
+
+.ZmTaskMultiView .DwtListView .RowDouble td {
+	@TaskListViewItemCheckbox@
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskSubject {
+	@TaskListViewItemSubject@
 }
 
-.newTaskBannerSep {
-	border-bottom:2px solid #AAA;
+.ZmTaskMultiView .DwtListView  .RowDouble .TaskDate {
+	@TaskListViewItemDate@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow {
+	@TaskListViewItemBottomContainer@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconOuterWrapper {
+	@TaskListViewItemBottomIconOuterWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .BottomRow .IconInnerWrapper {
+	@TaskListViewItemBottomIconInnerWrapper@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-Rows  .ImgCheckboxUnchecked-svg {
+	padding-top:32px;
+}
+
+.ZmTaskMultiView .DwtListView  .Row-selected .ImgCheckboxChecked-svg {
+	padding-top:32px;
```

Comment:
the above rule should be sufficient and this rule might be applying it again the same thing. (+.ZmTaskMultiView .DwtListView .DwtListView-Rows  .ImgCheckboxUnchecked-svg ).

Also, if the height of row is increased then the padding would not adjust automatically to vertically center this icon. Maybe if possible we can use display:table-cell or something ?